README.md

```diff
@@ -1,128 +1,201 @@
-# sneaky-package 🥸  
-A ready-to-deploy Python package designed to **stealthily integrate files** within a machine, ensuring discreet and seamless file operations without detection. This project is nothing more than a *lightweight & faster* Python package that aims to highlight and improve security by demonstrating potential vulnerabilities in the system when connected with **`pip`**.
+# SNEAKY PACKAGE 👾
+![GSSoC-ext](GSSoC-Ext.png)
+## Table of Contents
+1. [🚀Project Overview](#project-overview)
+2. [✨ Key Features ✨](#✨key-features✨)
+3. [⛓️ What is a **Pip Package**? ⛓️](#⛓️what-is-a-pip-package-⛓️)
+4. [🪪 Licence](#🪪licence)
+5. [🚀 How to Contribute to This Project](#🚀-how-to-contribute-to-this-project)
+   - [🍴 Fork the Repository](#🍴-fork-the-repository)
+   - [💻 Clone Your Fork](#💻-clone-your-fork)
+   - [🌿 Create a New Branch](#🌿-create-a-new-branch)
+   - [🛠️ Make Your Changes](#🛠️-make-your-changes)
+   - [✅ Finalize Your Changes](#✅-finalize-your-changes)
+   - [💬 Commit Your Changes](#💬-commit-your-changes)
+   - [📤 Push Your Changes](#📤-push-your-changes)
+   - [🔄 Create a Pull Request (PR)](#🔄-create-a-pull-request-pr)
+   - [🔎 Review Changes](#🔎-review-changes)
```

Comment:
Till here, all of this comes under how to proceed with contributing in the project. Give it a good heading and include all these points under it with git commands.